recipes-bsp/bootfiles/rpi-config.bb

```diff
@@ -36,6 +36,13 @@ GPIO_SHUTDOWN_PIN ??= ""
 
 inherit deploy nopackages
 
+RASPBERRYPI_CAMERA ??= "auto"
+RASPBERRYPI_CAMERA_V1 = "ov5647"
+RASPBERRYPI_CAMERA_V2 ?= "imx219"
+RASPBERRYPI_CAMERA_HQ = "imx477"
+RASPBERRYPI_CAMERA_GS = "imx296"
+RASPBERRYPI_CAMERA_V3 ?= "imx708"
```

Comment:
Any reason to have a mix of assignments and weak-assignments here?

Reply:
BTW, I like this approach.

Reply:
The weak assignment is to not override the retro compatibility with the previous RASPBERRYPI_CAMERA_V2/V3 = "1" flag and issue a warning instead.

Reply:
@vivien can you add a code comment for that? It will be very easy to miss in the future.